From: Mehdi Dogguy <mehdi@dogguy.org>
Date: Tue, 14 Mar 2017 09:50:01 +0100
Subject: Die instead of exit when filesystem/device is read-only

Dying gives a chance to NHC to apply appropriate actions before exiting.
--- a/nhc	2020-08-31 17:47:41.053541107 +0200
+++ b/nhc	2020-08-31 17:48:45.232575919 +0200
@@ -435,7 +435,7 @@
             echo "ERROR NHC:  FATAL:  Can't write $LOGFILE as $USER (uid $EUID) -- Read-only filesystem/device failure?"
             syslog "NHC:  FATAL:  Can't write $LOGFILE as $USER (uid $EUID) -- Read-only filesystem/device failure?"
             syslog_flush
-            exit 1
+            die 130 "Read-only filesystem/device failure?"	    
         else
             dbg "Output redirected per LOGFILE variable $LOGFILE"
             NHC_FD_OUT=3
